server/command.go

```diff
@@ -418,60 +418,17 @@ If you see an option to create a Jira issue, you're all set! If not, refer to ou
 	return p.responsef(header, addResponseFormat, jiraURL, p.GetSiteURL(), ji.GetMattermostKey(), pkey)
 }
 
-// executeUninstallCloud will uninstall the jira cloud instance if the url matches, and then update all connected
-// clients so that their Jira-related menu options are removed.
 func executeUninstallCloud(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
-	authorized, err := authorizedSysAdmin(p, header.UserId)
-	if err != nil {
-		return p.responsef(header, "%v", err)
-	}
-	if !authorized {
-		return p.responsef(header, "`/jira uninstall` can only be run by a System Administrator.")
-	}
-	if len(args) != 1 {
-		return p.help(header)
-	}
-
-	jiraURL, err := utils.NormalizeInstallURL(p.GetSiteURL(), args[0])
-	if err != nil {
-		return p.responsef(header, err.Error())
-	}
-
-	ji, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
-	if err != nil {
-		return p.responsef(header, "No current Jira instance to uninstall")
-	}
-
-	jci, ok := ji.(*jiraCloudInstance)
-	if !ok {
-		return p.responsef(header, "The current Jira instance is not a cloud instance")
-	}
-
-	if jiraURL != jci.GetURL() {
-		return p.responsef(header, "You have entered an incorrect URL. The current Jira instance URL is: `"+jci.GetURL()+"`. Please enter the URL correctly to confirm the uninstall command.")
-	}
-
-	err = p.instanceStore.DeleteJiraInstance(jci.GetURL())
-	if err != nil {
-		return p.responsef(header, "Failed to delete Jira instance "+ji.GetURL())
-	}
-
-	// Notify users we have uninstalled an instance
-	p.API.PublishWebSocketEvent(
-		wSEventInstanceStatus,
-		map[string]interface{}{
-			"instance_installed": false,
-			"instance_type":      "",
-		},
-		&model.WebsocketBroadcast{},
-	)
-
-	const uninstallInstructions = `Jira instance successfully disconnected. Go to **Settings > Apps > Manage Apps** to remove the application in your Jira Cloud instance.`
-
-	return p.responsef(header, uninstallInstructions)
+	return executeUninstall(p, c, header, "cloud", args...)
```

Comment:
1/5 change https://github.com/mattermost/mattermost-plugin-jira/blob/21a11bfc22191a399eefaee0813badcea1c7bfdc/server/command.go#L62-L63 to `"uninstall":    executeUninstall,` and check the parameter?

Reply:
Hi @levb , thanks for reviewing. That was in fact the first thing that came to my mind but I wasn't sure if just omitting the `/cloud|server` part there would automatically make them show up in the `args` slice. Is that the case?

Reply:
Yep, see https://github.com/mattermost/mattermost-plugin-jira/blob/master/server/command.go#L120